build-system/eslint-rules/dict-string-keys.js

```diff
@@ -42,7 +42,7 @@ function checkNode(node, context) {
     node.properties.forEach(function (prop) {
       if (!prop.key.raw && !prop.computed) {
         context.report({
-          node,
+          node: prop,
```

Comment:
Does this belong in a separate pull request?

Reply:
If that's preferred -- @jridgewell recently committed it to this PR because it is required for this to be able to pass the linter.